tracer/test/snapshots/StackExchangeRedisTests.Latest.verified.txt

```diff
@@ -0,0 +1,4940 @@
+[
```

Comment:
Wow, those are big snapshots :)

Reply:
Yeah, did you see the verification code I _removed_ though? 😛

Reply:
Yeah, it's going to be way simpler in the long run :D